internal/cmd/cli.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+)
+
+// CLI exposes common dependencies to commands.
+type CLI struct {
+	Stdin  io.Reader
+	Stdout io.Writer
+	Stderr io.Writer
+}
+
+// Output a string to CLI.Stdout. Output is like fmt.Printf except that it always
+// adds a trailing newline.
+// To write output without a trailing newline use CLI.Stdout directly.
+func (c *CLI) Output(format string, args ...interface{}) {
+	fmt.Fprintf(c.Stdout, format+"\n", args...)
+}
+
+// TODO indicates a place where os.Stdout is being used instead of CLI.Stdout.
+// It is temporary, and can be removed once everything is ported to use CLI
+// for output.
+var TODO = os.Stdout
```

Comment:
Were there some constraints you met that required adding this TODO or just trying to keep the size of the PR down?